### DISCUSSION TITLE
Errore in risposta alla richiesta di creazione di una fattura (elettronica)

### DISCUSSION THREAD

**Discussion by Brain4Web** [fattureincloud/api#285] [Q&A] [answered]
Buongiorno,
invio regolarmente, da inizio anno con la versione 2 delle api, le fatture generate da un servizio interno per iscrizione e rinnovo di servizi erogati dalla nostra azienda che sono poi inviate come fatture elettroniche.

Tali fatture (ad oggi più di 750) sono state create tramite la vs. piattaforma in modo corretto fino a fine Febbraio (le ultime il 24/02). Da ieri (01/03/2023) la piattaforma si rifiuta di crearle esibendo un errore 422 - "Il campo net_price è richiesto poiché use_gross_prices vale false.".

L'errore non sembra appropriato dato che il campo richiesto è presente nei dati della fattura (come lo è sempre stato) e il software di invio è il medesimo da inizio anno.

Ci sono state degli aggiornamenti al formato della creazione delle fatture che mi sono perso ?

Il nostro companyId è XXXXXXX.

Ivan

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Brain4Web 

dai vostri log vedo che state inserendo un oggetto _null_ all'interno dell'items_list. Credo che l'errore sia dovuto alla presenza di tale elemento.

```json
 "items_list": [
            null,
            {
                "description": "Fake item",
                "amount_net": 1111,
                "qty": 1,
                "vat": {
                    "id": 0
                }
            }
        ]
```
Vi chiederei di provare a rimuoverlo e riprovare (potete anche provare ad usare il JSON corretto da [API reference](https://developers.fattureincloud.it/api-reference)).

A disposizione
Mauro

**Reply by Brain4Web** [reply]
Ho verificato ed è vero.
Per un problema del gestionale che emette le fatture una delle due righe che compongono la fattura risulta "nulla".

Non avendo toccato nulla non ho verificato il passaggio precedente dandolo per corretto. Grosso errore...

Grazie mille.
